main.nf

```diff
@@ -459,30 +533,56 @@ process build_salmon_index {
     maxRetries 3
 
     input:
+        val(reduced) from REDUCED_SPLICI
         val(reduced) from REDUCED_CDNAS
-        tuple val(species), val(assembly), val(version), val(additionalTags) from CDNA_REFERENCE_FOR_SALMON
```

Comment:
Just to double check. You are removing this input (CDNA_REFERENCES_FOR_SALMON) in this process before deciding whether mode is 'normal' or 'splici'?

Reply:
thanks @nadjano Nevertheless, if it not clear at this point if `SPLICI_REFERENCE_FOR_SALMON` will be used as normal (cDNA) or splici, better to name this generically as `REFERENCE_FOR_SALMON` and then once this is decided use appropriate prefix